Examples/VimeoUpload-iOS/VimeoUpload-iOS/ViewControllers/VideoSettingsViewController.swift

```diff
@@ -285,8 +285,20 @@ class VideoSettingsViewController: UIViewController, UITextFieldDelegate
     
     private func applyVideoSettings()
     {
-        let videoUri = self.uploadTicket!.video!.uri!
-        let videoSettings = self.videoSettings!
+        guard let videoUri = self.uploadTicket?.video?.uri, let videoSettings = self.videoSettings else
+        {
+            let alertController = UIAlertController(
+                title: "Cannot Upload Video",
```

Comment:
I know the rest of the file doesn't do this yet, but can we move these strings into constants? Or a struct for constants?